src/messages/manifestjson.js

```diff
@@ -823,3 +823,12 @@ export const VERSION_FORMAT_INVALID = {
     https://mzl.la/3h3mCRu (MDN Docs) for more information.`),
   file: MANIFEST_JSON,
 };
+
+export const INCOGNITO_SPLIT_UNSUPPORTED = {
+  code: 'INCOGNITO_SPLIT_UNSUPPORTED',
+  message: i18n._('incognito "split" is unsupported.'),
+  description: i18n._(`The incognito "split" value is unsupported and will be
+    treated as "not_allowed" in Firefox. Remove the key from the manifest
+    file.`),
```

Comment:
Ee should not just recommend removal, but also point to documentation to explain how to develop the extension in a privacy-friendly way.

Since we don't have that yet (https://github.com/mdn/content/pull/33661/files#r1606120565), the next best alternative is to add a condition,


```suggestion
    file, if your extension is compatible with Firefox's private browsing mode.`),
```

(And maybe add a link to the incognito documentation? Any link is better than none?)